src/common/options/rgw.yaml.in

```diff
@@ -1913,6 +1913,14 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_max_copy_obj_concurrent_io
+  type: int
+  level: advanced
+  desc: the async refcount io number processed meanwhile in copy_obj
```

Comment:
Hello, thanks for contributing.  I would like to see this description be more clear.

@cbodley Does the below make sense?

"desc: Number of refcount operations to process concurrently when executing copy_obj"

Reply:
ok, this is much better.